app/src/main/java/org/simple/clinic/medicalhistory/newentry/NewMedicalHistoryScreen.kt

```diff
@@ -94,7 +94,7 @@ class NewMedicalHistoryScreen : BaseScreen<
   private val diabetesDiagnosis
     get() = binding.diabetesDiagnosis
 
-  private val questionViewEvents: Subject<NewMedicalHistoryEvent> = PublishSubject.create()
+  private val hotEvents: Subject<NewMedicalHistoryEvent> = PublishSubject.create()
```

Comment:
What does hotEvents mean?

Reply:
It's basically a stream that receives any `NewMedicalHistoryEvent`. For example for certain scenarios we cannot create a stream similar to button clicks or text changes, etc., So we just pass the event to this. This will be then added to `events` function.

Reply:
I understand the meaning, what I meant was the name is weirdly off.

Reply:
Yeah, it's something we are using in rest of the project 😅